c/chosen/chosen_rhel_8_4.sh

```diff
@@ -0,0 +1,31 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : chosen
+# Version       : 1.0.0
+# Source repo   : https://github.com/harvesthq/chosen/archive/1.0.0.zip
+# Tested on     : UBI 8
+# Script License: Apache License, Version 2 or later
+# Maintainer    : sachin.kakatkar@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#To Run: ./chosen_rhel_8_4.sh
+dnf install git wget npm make gcc-c++ bzip2 unzip ruby -y
```

Comment:
Avoid installing unwanted/unused dependencies.
Do we need `make, gcc-c++` for building this package ?

Reply:
Yes those packages are required.

Reply:
If you use the git hub repo (Github repo details can be found in below comment), you don't need `bzip2 unzip`.

I don't see any make command in the script .
I don't see any dependency on gcc and c++.

You only required, `git npm ruby`.